yarn.lock

```diff
@@ -1,1754 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We have a mix of NPM and Yarn usage currently. We should align on one (my vote is for yarn). Outside the scope of this PR ofc

Reply:
I removed yarn.lock as it seemed outdated and the package-lock.json file seemed more recently updated. :) 
https://github.com/mtennoe/swagger-typescript-codegen/pull/49

We can switch to yarn instead but it does not really matter too much I guess. (Maintaining two at least seems a bit of a hassle)